### DISCUSSION TITLE
Entering steam guard code, and lockouts

### DISCUSSION THREAD

**Discussion by ohhai-kthxbai** [mbround18/valheim-docker#189] [Q&A]
Firstly, Thanks for making this container!

When I setup the container/server using steam user and pass, I get many emails with the Steam guard code to authenticate. However, it attempts login so many times in quick success that steam locks further login attempts.

1. How do I stop it flooding the account with login attempts?
2. How do I enter my emailed guard code? Log says to run 'set_steam_guard_code' in container console but I tried that via portainer and the command isn't recognised. Also because of anti-flood I've only got seconds before the multiple requests mean that steam lock out my account!

Thanks!

**Comment by mbround18** [comment]
You should be able to use this repository without having to log into your steam user 0.o

**Reply by ohhai-kthxbai** [reply]
Thanks. I do understand that. 
I want to make it work, but can't!

**Reply by CosmicHorrorDev** [reply]
You're sure you're using the docker image from this repository? All the logins being used are anonymous, so no credentials should be required.

Can you also attach a screenshot of what you're talking about (without including your username or password or any of that)?
